tests/rules/javascript/test.content_scripts_file_absent.js

```diff
@@ -15,10 +15,8 @@ function createJsScanner(code, validatedFilename, existingFiles = {}) {
 describe('content_scripts_file_absent', () => {
   it('should show an error when content script is missing', async () => {
     const nonExistentFiles = [
```

Comment:
Nit, with just one scenario to test here, I'm not sure that it is still worth that this test still iterates on a `nonExistentFiles` array, we could just remove the `nonExistentFiles.forEach` and the `nonExistentFiles` and make this test a bit simpler to read.

Reply:
Fixed in 9e67fa4